src/iconc/ccomp.c

```diff
@@ -173,11 +173,15 @@ char *srcname;
 char *exename;
    {
    struct lib *l;
-   char sbuf[MaxFileName];		/* file name construction buffer */
    int  rv, buflen;
-   char *buf, objname[MaxFileName];
+   char *buf;
    char *s;
-   char *dlrgint;
+   /* dlrgint disabled for now */
+#if 0
```

Comment:
I started to mark places like this with TODO or FIXME for easy grepping. Some IDEs also know how track those labels